net.sourceforge.kmetronome.json

```diff
@@ -0,0 +1,79 @@
+{
+  "app-id": "net.sourceforge.kmetronome",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "kmetronome",
+  "rename-icon": "kmetronome",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
+    /* MIDI */
+    "--device=all",
+    /* Allow loading, saving files from anywhere (portals don’t work yet) */
+    "--filesystem=home",
```

Comment:
does a metronome really need to access the whole home directory?

Reply:
The user may create pattern files any where, and they should be reloaded automatically upon startup (i.e. not only interactive loaded on demand)